tests/src/core/AnimatorHang.test.ts

```
@@ -0,0 +1,20 @@
+canvasDOM.width = 1024;
+canvasDOM.height = 1024;
+describe("Canvas 1024 test", async function () {
+  const engine = await WebGLEngine.create({ canvas: canvasDOM });
+  const scene = engine.sceneManager.activeScene;
+  const rootEntity = scene.createRootEntity();
+  rootEntity.addComponent(Camera);
+  const resource = await engine.resourceManager.load<GLTFResource>(glbResource);
+  const defaultSceneRoot = resource.defaultSceneRoot;
+  rootEntity.addChild(defaultSceneRoot);
+  const animator = defaultSceneRoot.getComponent(Animator);
+  it("loaded", () => { expect(animator).not.eq(null); });
```

ESLint

[error] Replace `·expect(animator).not.eq(null);` with `⏎····expect(animator).not.eq(null);⏎·`

(prettier/prettier)